tests/phpunit/tests/html-api/wpHtmlProcessorBreadcrumbs.php

```diff
@@ -37,22 +37,39 @@ public function test_navigates_into_normative_html_for_supported_elements( $html
 	public function data_single_tag_of_supported_elements() {
 		$supported_elements = array(
 			'A',
+			'ADDRESS',
+			'ARTICLE',
+			'ASIDE',
 			'B',
 			'BIG',
 			'BUTTON',
+			'CENTER', // Neutralized
 			'CODE',
+			'DETAILS',
+			'DIALOG',
```

Comment:
Are we missing `DIR`?

```suggestion
			'DIALOG',
			'DIR',			
```

Reply:
We were; thanks for noticing this! It's added now.